examples/Python/python-django-webapp-sample/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django
+whitenoise
+azure-appconfiguration-provider
```

Comment:
Can you set a required version number. 1.0.0b2. This will not work with any other version. You already have this with Flask.

Reply:
True. This has been updated. Thanks!